src/docs/product/error-monitoring/states-triage/index.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Issue States and Triage
+sidebar_order: 30
+description: "Learn more about issue states and how to triage issues."
+---
+
+An issue can be in one of three states:
+
+1. Unresolved: The default state for all new issues.
+2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
+3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.
+
+The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:
+
+- A new issue is created; it's labeled "New Issue"
+- A resolved issue re-occurs; it's labeled "Regression"
+- An ignored issue meets its ignore conditions; it's labeled "Unignored"
+
+For-review issues are a subset of all unresolved issues.
+
+## Issue Triage
+
+Triaging is the act of deciding what action you want to take on an issue. It typically involves one or more of these actions.
+
+### Assign
+
+You can assign issues to teams or individuals. In a multi-team environment, routing issues to the right people usually starts with assigning it to the right team, which further assigns it to the right member of the team. [Ownership rules](/product/error-monitoring/issue-owners/) allow you to define rules for which teams or individuals should own an issue. You might also see a list of suggested assignees including the owners of suspect commits.
+
+TO DO Confirm meaning in first sentence; does someone have to manually do this reassigning to a team member after it gets assigned to a team?
+
+### Link
```

Comment:
Can we call this `Linked Issues`?

Reply:
I suggest "Link Issues" because all of these titles are actions you can take on an issue and we want to maintain parallel structure.